ci-operator/jobs/openshift/sdn/openshift-sdn-master-presubmits.yaml

```diff
@@ -1,321 +1,5 @@
 presubmits:
   openshift/sdn:
```

Comment:
when will this file go away? given there are no more jobs in sdn repo moving forward?

Reply:
When we move the kube-proxy build somewhere else. (Or possibly just stop building it.)